V2/tests/api/azure/project_azure_endpoints_test.py

```diff
@@ -0,0 +1,254 @@
+import json
+from http import HTTPStatus
+
+import pytest
+from faker import Faker
+import azure.functions as func
+
+from time_tracker.projects._application import _projects as azure_projects
+from time_tracker.projects import _domain as domain
+from time_tracker.projects import _infrastructure as infrastructure
+
+PROJECT_URL = '/api/projects/'
+
+
+@pytest.fixture(name='insert_project')
+def insert_project(test_db, insert_customer, project_factory, customer_factory) -> domain.Project:
+    inserted_customer = insert_customer(customer_factory(), test_db)
+
+    def _new_project():
+        project_to_insert = project_factory(customer_id=inserted_customer.id)
+        dao = infrastructure.ProjectsSQLDao(test_db)
+        inserted_project = dao.create(project_to_insert)
+        return inserted_project
+    return _new_project
+
+
+def test__project_azure_endpoint__returns_all_projects(
+    insert_project
+):
+    inserted_projects = [
+        insert_project().__dict__,
+        insert_project().__dict__
+    ]
+
+    req = func.HttpRequest(method='GET', body=None, url=PROJECT_URL)
+    response = azure_projects._get_projects.get_projects(req)
+    projects_json_data = response.get_body().decode("utf-8")
+
+    assert response.status_code == HTTPStatus.OK
+    assert projects_json_data == json.dumps(inserted_projects)
+
+
+def test__project_azure_endpoint__returns_an_project__when_project_matches_its_id(
+    insert_project
+):
+    inserted_project = insert_project().__dict__
+
+    req = func.HttpRequest(
+        method='GET',
+        body=None,
+        url=PROJECT_URL,
+        route_params={"id": inserted_project["id"]},
+    )
+
+    response = azure_projects._get_projects.get_projects(req)
+    activitiy_json_data = response.get_body().decode("utf-8")
+
+    assert response.status_code == HTTPStatus.OK
+    assert activitiy_json_data == json.dumps(inserted_project)
+
+
+def test__projects_azure_endpoint__returns_a_status_code_400__when_project_recive_invalid_id(
+):
+    req = func.HttpRequest(
+        method="GET",
+        body=None,
+        url=PROJECT_URL,
+        route_params={"id": "invalid id"},
+    )
+
+    response = azure_projects._get_projects.get_projects(req)
+
+    assert response.status_code == HTTPStatus.BAD_REQUEST
+    assert response.get_body() == b"Invalid Format ID"
+
+
+def test__project_azure_endpoint__returns_an_project_with_inactive_status__when_a_project_matching_its_id_is_found(
```

Comment:
The name of test test__project_azure_endpoint__returns_a_project_with_inactive_status__when_a_project_matching_its_id_is_found with a inside an